static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -242,7 +245,7 @@ function TagFacetsDistributionMeter({
 
   return (
     <TagSummary>
-      <details open={expanded} onClick={e => e.preventDefault()}>
+      <details open onClick={e => e.preventDefault()}>
```

Comment:
Yeah, I had to force `open` to be true otherwise the collapse animation doesn't really work. When the `expanded` state changes, the animation is supposed to kick off but due to `open={expanded}` the element just gets hidden in the dom immediately and doesn't animate.

So with this change, the actual display of the legend container is controlled by via framer-motion now